test/metric_value_benchmark/extra_configs/collectd.conf

```diff
@@ -0,0 +1,12 @@
+LoadPlugin network
```

Comment:
This would be the [same reasons with the above questions though ](https://github.com/aws/amazon-cloudwatch-agent-test/pull/13#discussion_r1035376992). 
```
to limit the amount of metrics we set and less rely on the default enable plugins from collectd, it would better to setup the collectd configuration, setup all these steps beforehand and provide more flexibility when testing collectd
```

Reply:
This doesn't explain why we can't configure collectd in the image build.

Reply:
In more simpler way, would it be better with `provide more flexibility when testing collectd` ? e.g testing with different plugins with collectd ? If that reason is not convinced enough, then I'm fine with configure collectd in the image build.